doc/nrf/known_issues.rst

```diff
@@ -253,6 +247,14 @@ NCSDK-10106: Elevated current consumption when using applications without :ref:`
 
   **Workaround:** Enable :ref:`nrfxlib:nrf_modem`.
 
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+CIA-351: Connectivity issues with Azure IoT Hub
+  If a ``device bound`` message is sent to the device while the device is in the LTE Power Saving Mode, the TCP connection will most likely be terminated by the server.
+  Known symptoms of this are frequent reconnections to cloud, messages sent to Azure IoT Hub never arriving, and FOTA images being downloaded twice.
+
+  **Workaround:** Avoid using LTE Power Saving Mode and extended DRX intervals longer than approximately 30 seconds. This will in general reduce the risk of the issue occurring, at the cost of increased power consumption.
```

Comment:
```suggestion
  **Workaround:** Avoid using LTE Power Saving Mode and extended DRX intervals longer than approximately 30 seconds. This will reduce the risk of the issue occurring, at the cost of increased power consumption.
```